core/ops_json.rs

```diff
@@ -59,15 +42,20 @@ pub fn json_op_sync<F, V, R>(op_fn: F) -> Box<OpFn>
 where
   F: Fn(&mut OpState, V, &mut [ZeroCopyBuf]) -> Result<R, AnyError> + 'static,
   V: DeserializeOwned,
-  R: Serialize,
+  R: Serialize + 'static,
 {
-  Box::new(move |state: Rc<RefCell<OpState>>, mut bufs: BufVec| -> Op {
-    let result = serde_json::from_slice(&bufs[0])
-      .map_err(AnyError::from)
-      .and_then(|args| op_fn(&mut state.borrow_mut(), args, &mut bufs[1..]));
-    let buf =
-      json_serialize_op_result(None, result, state.borrow().get_error_class_fn);
-    Op::Sync(buf)
+  Box::new(move |state, payload, buf: OpBuf| -> Op {
+    // For sig compat map OpBuf to BufVec
+    let mut bufs: BufVec = match buf {
+      Some(b) => vec![b],
```

Comment:
What not keep using SmallVec?

Reply:
Virtually no ops used variadic buffers, they had one control buffer and an optional data buffer, control buffer is now superseded by a control/structured arg.

Removing the unnecessary variadicity is cleaner IMO, it's best to drop it if it's not used and also variadic args would be inherently incompatible with v8 Fast API so would hinder moving bin/io ops over, even if it supported typed integer arrays.

This code is just a stopgap till follow-up PR `1.`:
> 1. Change op-sig, removing variadic BufVec

Reply:
OK